### ISSUE TITLE
Used this but something didnt work

### ISSUE THREAD

**Issue by googggg** [CheeseDanish1/Rosetta-Stone-Script#20]
so i used this and submitted it however my teacher wrote back and said "I see that you have scored high on each lesson, but they remain incomplete. Can you please go back and make sure you click right until the end to make sure that each lesson is complete so I can give you a final grade. Merci!"

what should i do, 
i went back to the lessons i used this on and when i try to do a lesson its like as if i never did it

**Comment by clownavision** [comment]
gaslight your teacher

**Comment by googggg** [comment]
> gaslight your teacher

I DID AND IT WORKED LMAOO

**Comment by jimmybenoit** [comment]
> > gaslight your teacher
> 
> I DID AND IT WORKED LMAOO

damn gg

**Comment by jimmybenoit** [comment]
I'm using this right now but I'm afraid my teacher will see something is up because for some reason the time complete always comes up as 8 minutes. Did you have problems with that?

**Comment by googggg** [comment]
oh shit mb i didnt see this, nah i didnt have a problem with it

**Comment by Kyzo-pixel** [comment]
> oh shit mb i didnt see this, nah i didnt have a problem with it, or she didnt mention it

did u used fluency builder ?

**Comment by googggg** [comment]
nah brightspace

**Comment by FlameWick** [comment]
yeah for me they are all saying 100% but they arent showing as completed in the percentage
